packages/nextjs/src/config/webpack.ts

```diff
@@ -435,6 +438,57 @@ async function addSentryToClientEntryProperty(
   return newEntryProperty;
 }
 
+/**
+ * Make sure the instrumentation file has a `onRequestError` Handler
+ *
+ * @param projectDir The root directory of the project, where config files would be located
+ */
+function warnAboutMissingonRequestErrorHandler(projectDir: string): void {
+  const instrumentationPaths = [
+    ['src', 'instrumentation.ts'],
+    ['src', 'instrumentation.js'],
+    ['instrumentation.ts'],
+    ['instrumentation.js'],
+  ];
```

Comment:
Looks like this approach does not respect `pageExtensions` in Next.js config. I created https://github.com/getsentry/sentry-javascript/issues/15652 to track this bug.